wire-schema/src/main/java/com/squareup/wire/schema/SchemaLoader.java

```diff
@@ -156,9 +160,29 @@ private Schema loadFromDirectories(Map<Path, Path> directories) throws IOExcepti
       }
     }
 
+    ProtoFile descriptor = loaded.get(DESCRIPTOR_PROTO);
+    if (descriptor == null) {
+      loaded.put(DESCRIPTOR_PROTO, loadDescriptorProto());
+    }
+
     return new Linker(loaded.values()).link();
   }
 
+  /**
+   * Returns Google's protobuf descriptor, which defines standard options like default, deprecated,
+   * and java_package. If the user has provided their own version of the descriptor proto, that is
+   * preferred.
```

Comment:
Can we just require that it be included in the set when running the parser? It seems strange to pull a copy from somewhere other than what's being supplied.